src/ui/public/stringify/types/Url.js

```diff
@@ -1,6 +1,13 @@
 define(function (require) {
   return function UrlFormatProvider(Private, highlightFilter) {
     var _ = require('lodash');
+    require('!!file?name=[path][name].[ext]!ui/stringify/icons/go.png');
```

Comment:
Would it make more sense for a maintainability standpoint to put these require statements into their own file, ie: 
``` 
src/ui/public/stringify/icons/index.js
```
and then include that file here?